### PR TITLE
[Improved] Clone error message suggests networking might be involved

### PR DESCRIPTION
**Pull request by agisilaos**
Fixes: #1789

With this PR the error message related to updating a clone repo is now improved with the suggestions of @shiftkey. Happy to improve it even more. 

P.s It seems like that I'm into fixing and improving error messages 😂😅. Btw it's a great way to explore and learn the codebase a bit more. ❤️

### CODE REVIEW

File: app/src/ui/clone-repository/clone-repository.tsx [hunk 1]
```diff
@@ -344,7 +344,7 @@ export class CloneRepository extends React.Component<
 
     if (!url) {
       const error = new Error(
-        `We couldn't find that repository. Check that you are logged in, and the URL or repository alias are spelled correctly.`
+        `We couldn't find that repository. Check that you are logged in, the network is accessible and the URL or repository alias are spelled correctly.`
```
Comment (j-f1) [inline app/src/ui/clone-repository/clone-repository.tsx]: > ...the network is accessible<b><u>,</u></b> and the URL...

Comma please? 😄

Comment (agisilaos) [reply app/src/ui/clone-repository/clone-repository.tsx]: Instead of the word `and`? @j-f1

Comment (j-f1) [reply app/src/ui/clone-repository/clone-repository.tsx]: Nope, just what you did just now in https://github.com/desktop/desktop/pull/2872/commits/2bd1c9cc68723205c0fbf1da9fed4dcaf8937c06.

Comment (agisilaos) [reply app/src/ui/clone-repository/clone-repository.tsx]: Pefect! I thought so. Thanks for the comment 💯

**Comment by agisilaos** [comment]
I think it's ready to be reviewed again @j-f1. What you think?

**Pull request merged**